bin/network-monitor/assets/index.html

```diff
@@ -147,10 +153,39 @@
             servicesCount.textContent = `${totalServices} Services`;
 
             // Generate status cards
-            container.innerHTML = processedServices.map(service => {
+            const serviceCardsHtml = processedServices.map(service => {
                 const isHealthy = service.status === 'Healthy';
                 const statusColor = isHealthy ? '#22C55D' : '#ff5500';
                 const statusIcon = isHealthy ? '✓' : '✗';
+                const numOrDash = value => isHealthy ? (value?.toLocaleString?.() ?? value ?? '-') : '-';
+                const timeOrDash = ts => {
+                    if (!isHealthy) return '-';
+                    return ts ? new Date(ts * 1000).toLocaleString() : '-';
+                };
+                const commitmentOrDash = (value, label) => isHealthy && value
+                    ? `
+                        ${value.substring(0, 20)}...
+                        <button class="copy-button" onclick="copyToClipboard('${value}', event)" title="Copy full ${label}">
+                            <svg class="copy-icon" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2">
+                                <rect x="9" y="9" width="13" height="13" rx="2" ry="2"></rect>
+                                <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1"></path>
+                            </svg>
+                        </button>
```

Comment:
This is the same copy button as in other instances, right? If so, I wonder if it is worth creating a function that generates it.